www/shared/components/cookie-banner/CookieBanner.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+const mockCookieConsents = [];
+const mockAdd = jest.fn().mockImplementation((cookie) => mockCookieConsents.push(cookie));
+
+jest.doMock('@enzsft/react-cookie-consents',
+    () => ({
+        useCookieConsents: () => ({
+            get: () => mockCookieConsents,
+            add: mockAdd,
+        }),
+    }),
+);
+
+it('should render banner when cookie consent was not accepted', () => {
+    const CookieBanner = require('./CookieBanner').default;
```

Comment:
This is repeated in each test, can't it be defined outside the tests?

Reply:
Yes! I was trying to do something previously that needed those requires but i dont need them anymore 😬fixed!